pyodide_build/cli/config.py

```diff
@@ -47,7 +49,7 @@ def get_config(
     configs = _get_configs()
 
     if config_var not in configs:
-        typer.echo(f"Config variable {config_var} not found.")
-        typer.Exit(1)
+        print(f"Config variable {config_var} not found.", file=sys.stderr)
```

Comment:
What's the purpose of changing typo.echo to print? If the goal is to avoid using typer APIs we might change it everywhere.

Reply:
The point is to print to stderr.

Reply:
The docs at https://typer.tiangolo.com/tutorial/printing/?h=stderr#printing-to-standard-error say that we need to use `stderr=True`, but it has to be done at the `rich.console.Console()` level. Maybe we should set that somewhere?